LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 Grakn Labs.
+   Copyright 2021 Grakn Labs.
```

Comment:
Is this expected @haikalpribadi ? We've got an Apache `LICENSE` file, and we're also embedding a copyright statement in it.

Reply:
I don't know @lolski - this looks weird. Do we have it on other Apache license? I don't think so, right? If so we should remove it.